ThunderRequest/TSCRequestController.m

```diff
@@ -608,7 +608,7 @@ - (void)scheduleDownloadRequest:(TSCRequest *)request on:(NSDate *)beginDate pro
 			NSURLRequest *normalisedRequest = [self backgroundableRequestObjectFromTSCRequest:request];
 			NSURLSessionDownloadTask *task = [welf.backgroundSession downloadTaskWithRequest:normalisedRequest];
             
-            if (@available(iOS 11.0, *)) {
+            if (@available(iOS 11.0, *) && @available(watchOS 4.0, *)) {
```

Comment:
You should be able to make this `if @available(iOS 11.0, watchOS 4.0, *) {`

Reply:
Just came to the same conclusion! ❤️